packages/scripts/src/create-entrypoint.ts

```diff
@@ -105,89 +81,66 @@ export function createMainEntrypoint({
     publicPathVariableName,
     configurations,
     mode,
-    staticBuild,
+    // staticBuild, //TODO: check all places that replaced this flow
     publicConfigsRoute,
-    config,
+    config = [],
     target,
     eagerEntrypoint,
     env,
     featuresBundleName,
     configLoaderModuleName = '@wixc3/engine-scripts/dist/default-config-loader',
 }: ICreateEntrypointsOptions) {
-    const envName = env.name;
-    const configs = getAllValidConfigurations(getConfigLoaders(configurations, mode, configName), envName);
-
+    const configs = getAllValidConfigurations(getConfigLoaders(configurations, mode, configName), env.name);
+    const engineCoreSpecifier = JSON.stringify(require.resolve('@wixc3/engine-core'));
```

Comment:
remove this. we want to avoid absolute paths in generate code, so bundle will be reproducible